spec/features/task_lists_spec.rb

```diff
@@ -51,7 +51,6 @@ def visit_issue(project, issue)
 
       expect(page).to have_selector(container)
       expect(page).to have_selector("#{container} .wiki .task-list .task-list-item .task-list-item-checkbox")
-      expect(page).to have_selector("#{container} .js-task-list-field")
```

Comment:
These three removed expectations were there for a reason -- they're required for task lists to work.